src/cascadia/TerminalApp/userDefaults.json

```diff
@@ -6,6 +6,12 @@
 
     "defaultProfile": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
 
+    // If enabled, selections are automatically copied to your clipboard
+    // To learn more about global settings, visit https://aka.ms/terminal-global-settings
+    "copyOnSelect": false,
+
+    // Profiles are the list of command line executables in your dropdown

```

Comment:
This still isn't fully right. @DHowett do you have an idea of how to word this?

Reply:
I think you mean his twin, @DHowett-MSFT

Reply:
![](https://media.giphy.com/media/iHI0ef8D3jaDK/giphy.gif)

Reply:
What about something like:
"Profiles are listed in your dropdown and they include the command line executable along with customizations."